browser.go

```diff
@@ -84,6 +84,20 @@ func (b *browserImpl) NewContext(options ...BrowserNewContextOptions) (BrowserCo
 	context := fromChannel(channel).(*browserContextImpl)
 	context.browser = b
 	b.browserType.(*browserTypeImpl).didCreateContext(context, &option, nil)
+	if os.Getenv("PW_FREEZE_TIME") != "" {
```

Comment:
I'd remove this, since we are testing this kind of stuff only in the microsoft/playwright repository.

Reply:
It's been removed.  And I need to ask you to merge this PR, which I can't due to `macos-13` in CI is switched back to `macos-latest`.